conans/client/command.py

```diff
@@ -1191,10 +1191,11 @@ def search(self, *args):
         If you provide a pattern, then it will search for existing package
         recipes matching it.  If a full reference is provided
         (pkg/0.1@user/channel) then the existing binary packages for that
-        reference will be displayed.  If no remote is specified, the search
-        will be done in the local cache.  Search is case sensitive, exact case
-        has to be used. For case insensitive file systems, like Windows, case
-        sensitive search can be forced with '--case-sensitive'.
+        reference will be displayed. There is no default remote, if no remote
```

Comment:
Actually "there is no default remote" is confusing, because the first remote is the default, but that doesn't mean that it is the default for searching

Reply:
So remove the prepend and postpend "cache without the default remote" ?